wallet/dapps-communication.md

```diff
@@ -411,9 +416,246 @@ required:
  "publicKey": "P12..."
 }
 ```
+</td>
+</tr>
+</tbody>
+</table>
+
+#### Rolls
+
+##### Sell
+
+<table>
+<thead>
+<tr>
+<th>Direction</th>
+<th>Type</th>
+<th>Format</th>
+<th>Example</th>
+</tr>
+</thead>
+<tbody>
+
+<tr>
+<td>Webpage to extension</td>
+<td><code>account.sellRolls</code></td>
+<td>
+
+```yaml
+type: object
+properties:
+  fee:
```

Comment:
what is the purpose of fee in buy/sell roll? where is the best place to document this?

Reply:
The fee is define the amount of fee to be used in order to speed up the transfer (buy/sell). Its widely used in web3 too.